### PR TITLE
Fix dump effective bandwidth

### PR DESCRIPTION
**Pull request by zasdfgbnm**
IIUC, all inputs, outputs, and global buffers are pushed to `args`, so computing output bytes separately causes a double-counting problem.

### CODE REVIEW

**Review by jjsjann123 — APPROVED** [review]
Oops, good catch.

Since we are at this topic, we would probably  want to count the IO buffers twice though.

File: torch/csrc/jit/codegen/cuda/executor.cpp [hunk 1]
```diff
@@ -1260,7 +1262,7 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
 
     bytes_processed_ = 0;
     // Figure how many bytes are inputs, outputs, and temporary buffers
-    for (auto i : c10::irange(args.size())) {
+    for (auto i : c10::irange(num_inputs)) {
```
Comment (zasdfgbnm) [inline torch/csrc/jit/codegen/cuda/executor.cpp:1265]: cc @naoyam

Comment (naoyam) [reply torch/csrc/jit/codegen/cuda/executor.cpp:1265]: Thanks. I believe some code cleanup would be due here (not about this particular fix)

**Review by naoyam — APPROVED** [review]

File: torch/csrc/jit/codegen/cuda/executor.cpp (hunk 1 above)
Comment (jjsjann123) [reply torch/csrc/jit/codegen/cuda/executor.cpp:1265]: Apology for this half finished work. I get dragged away in the middle of the refactor.

I do plan on coming back here and clean things up after 1.13.

Comment (csarofeen) [reply torch/csrc/jit/codegen/cuda/executor.cpp:1265]: whoops 😳

**Pull request merged**